test/atmosphere_model_unit.jl

```diff
@@ -43,11 +43,11 @@ end
     set!(θᵢ, (x, y, z) -> θ₀ + rand())
     set!(model; θ=θᵢ)
 
-    θ_model = Breeze.AtmosphereModels.PotentialTemperatureField(model)
+    θ_model = potential_temperature(model) |> Field
```

Comment:
`potential_temperature` doesn't exist.  Was it meant to be
```suggestion
    θ_model = liquid_ice_potential_temperature(model) |> Field
```
?